### PR TITLE
Api connecting

### PR DESCRIPTION
**Pull request by elleneub**

### CODE REVIEW

File: APICalls.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- returns the next problem data
+requestNextProblem(level: Integer, studentID: String)
+handleNextProblem(problemData: String)
+
+/addProblemAttempt/<start_time>/<end_time>/<answer>/<wasCorrect>
+- add problem history to database
+attemptAddProblemData(start_time: TypeTBD, end_time: TypeTBD, answer: String, wasCorrect: Bool)
+handleAddProblemDataAttempt(added: Bool)
+
+
```
Comment (zoefarrell) [inline APICalls.txt:13]: comment

**Review by zoefarrell — CHANGES_REQUESTED** [review]
looks great!

File: APICalls.txt (hunk 1 above)
Comment (elleneub) [reply APICalls.txt:13]: asdf

**Review by elleneub — COMMENTED** [review]
Fixed it

**Pull request merged**